server/initializers/updates.coffee

```diff
@@ -22,14 +22,33 @@ checkUpdate = (notifier, app) ->
             log.raw err
         else if setUpdate
             log.info "#{app.name} - update required."
-            messageKey = 'update available notification'
-            message = localization.t messageKey, appName: app.name
-            notifier.createTemporary
-                text: message
-                resource: app: 'home'
         else
             log.info "#{app.name} - no update required."
 
+        shouldBeUpdated = not err? and setUpdate
```

Comment:
Me too, I figured that since there are no "error" parameter in the callback, it looks more semantic.